### DISCUSSION TITLE
How to show Guides, Tutorials and so forth

### DISCUSSION THREAD

**Discussion by clfest** [davidgranstrom/scnvim#218] [Q&A]
How can I access the Guides, Tutorials and have them properly displayed in scnvim? For example: 'OSC communication'.

                                     Thank You For Your Attention

**Comment by gilfuser** [comment]


**Reply by clfest** [reply]
I don't think so. My computer with scnvim is a RPi 400, where Qt doesn't work so no complete SCIDE. On my other computer it does work, so 'OSC communication' can be displayed as a page.

In the scnvim plugin :SCNvimHelp SinOsc decodes /usr/local/share/SuperCollider/HelpSource/Classes/SinOsc.schelp (I think) and displays it on the screen through pandoc, however I can't get /usr/local/share/SuperCollider/HelpSource/Guides/OSC_communication.schelp to display in scnvim, Similarly (I think) Guides like /usr/local/share/SuperCollider/HelpSource/Tutorials/JITLib/jitlib_basic_concepts_01.schelp and so forth.

                                                              Thank You For Your Attention

**Reply by gilfuser** [reply]
You are right! I deleted my previous comment to avoid future confusion. I mixed things up. Sorry.
Personally I'm using :SCNvimHelp, but if you don't have QT I don't know how to do it. Perhaps there is a way here:
https://github.com/davidgranstrom/scnvim/wiki/Tips-and-tricks#jump-to-the-source-of-a-help-file

I used the SuperCollider help system as you described, but that was before it was rebuilt into Lua. Not sure if this feature made it through the portal.
